pkg/apiserver/statement/service.go

```diff
@@ -195,7 +204,12 @@ func (s *Service) plansHandler(c *gin.Context) {
 		return
 	}
 	db := utils.GetTiDBConnection(c)
-	plans, err := QueryPlans(db, req.BeginTime, req.EndTime, req.SchemaName, req.Digest)
+	schema, err := getStatementSchema(db)
```

Comment:
How about caching this in `service` instead of passing it everywhere?

Reply:
Thanks for the tip, this is very helpful!